### PR TITLE
Factor out Visualization#addClaimElements().

### PR DESCRIPTION
**Pull request by FrankHeijden**
* Makes #addClaimElements() public

### CODE REVIEW

**Comment by FrankHeijden** [comment]
Can't do anything about the diff. Please refer to your IDE or diff from the CLI for a proper diff.

**Comment by FrankHeijden** [comment]
That seems to have fixed the problem.

File: src/main/java/me/ryanhamshire/GriefPrevention/Visualization.java [hunk 1]
```diff
@@ -130,23 +132,11 @@ public static Visualization FromClaim(Claim claim, int height, VisualizationType
     //handy for combining several visualizations together, as when visualization a top level claim with several subdivisions inside
     //locality is a performance consideration.  only create visualization blocks for around 100 blocks of the locality
 
-    private void addClaimElements(Claim claim, int height, VisualizationType visualizationType, Location locality)
+    public void addClaimElements(Claim claim, int height, VisualizationType visualizationType, Location locality)
```
Comment (RoboMWM) [inline src/main/java/me/ryanhamshire/GriefPrevention/Visualization.java:135]: Is this the only logic change?

Comment (RoboMWM) [reply src/main/java/me/ryanhamshire/GriefPrevention/Visualization.java:135]: mk, would flip title and description IMO.

**Review by RoboMWM — APPROVED** [review]

**Pull request merged**